client/src/charts/legends/LegendList.tsx

```diff
@@ -6,25 +6,20 @@ import { CheckBox, CheckBoxProps } from "src/components/CheckBox/CheckBox";
 
 import { is_a11y_mode } from "src/core/injected_build_constants";
 
-import "./LegendList.scss";
+import type { LegendItemType } from "./LegendItemType";
 
-interface LegendItemProps {
-  active: boolean;
-  id: string | number;
-  label: string;
-  color?: string;
-}
+import "./LegendList.scss";
 
-interface LegendListProps {
+export interface LegendListProps {
   isHorizontal?: boolean;
-  items: LegendItemProps[];
+  items: LegendItemType[];
   onClick: Function;
```

Comment:
Haven't looked at this too closely, but I think this should be something like `onClick: (event: React.MouseEvent) => void`

Reply:
Or `onClick: React.ReactEventHandler<element>`?